packages/slotted-items-mixin/README.md

```diff
@@ -0,0 +1,9 @@
+# @vaadin/slotted-items-mixin
+
+This package provides `SlottedItemsMixin` that does the following:
+
+1. Set read-only `items` property based on the light DOM children
+
+2. Add `slotchange` listener on the default `slot` to update items
+
+3. Dispatch `items-changed` event when `items` value 
```

Comment:
Something is missing at the end. Maybe something like "Dispatch `items-changed` event when `items` value is changed"?

Reply:
Done.